README.md

```diff
@@ -29,33 +29,33 @@
 ## Installation
 
 ```bash
-$ npm install
+$ pnpm install
 ```
 
 ## Running the app
 
 ```bash
 # development
-$ npm run start
+$ pnpm run start
```

Comment:
Can you please remove `run` from all the examples? I think it's not needed with `pnpm`.